src/main/java/pl/uj/io/cuteanimals/model/action/ShowStats.java

```diff
@@ -0,0 +1,52 @@
+package pl.uj.io.cuteanimals.model.action;
+
+import java.util.List;
+import pl.uj.io.cuteanimals.model.GameState;
+import pl.uj.io.cuteanimals.model.PlayerAttributes;
+import pl.uj.io.cuteanimals.model.Result;
+import pl.uj.io.cuteanimals.model.interfaces.IAction;
+import pl.uj.io.cuteanimals.model.interfaces.ICharacter;
+import pl.uj.io.cuteanimals.model.interfaces.IResult;
+
+public class ShowStats implements IAction {
+    @Override
+    public IResult execute(ICharacter character) {
+        if (!getAcceptableStates().contains(character.getCurrentGameState())) {
+            return new Result("This action cannot be executed now");
```

Comment:
Maybe change to sth like this:
```suggestion
            return new Result("This isn't time for this.");
```
Just to keep immersion.